db/models.py

```diff
@@ -1 +1,28 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.CharField(max_length=255, blank=True, null=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.TextField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+    guild = models.ForeignKey(Guild,
+                              on_delete=models.SET_NULL,
+                              blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
You've correctly used `auto_now_add=True` for the `created_at` field in the `Player` model. This will automatically set the field to the current date and time when a Player object is created, which is typically the desired behavior for a created_at field.